Controller and service creation

AdanAlvarezV opened the pull request:
Created service and controller with get all persons and create new person

AdanAlvarezV committed:
Controller and service creation

Created service and controller with get all persons and create new person

simontorres123 commented on src/main/java/com/demo/unittesting/service/IPersonService.java:
```diff
@@ -0,0 +1,14 @@
+@Service
+public interface IPersonService {
+
+    public List<Person> getAllPerson();
```
you can avoid to add access modifier "public" because by deafult it is public

AdanAlvarezV committed:
Removed public access modifier

Removed public access modifier

simontorres123 commented on src/main/java/com/demo/unittesting/service/PersonServiceImpl.java:
```diff
@@ -0,0 +1,23 @@
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
```
add @Service at implementation

simontorres123 replied:
done

simontorres123 commented on src/main/java/com/demo/unittesting/service/IPersonService.java:
```diff
@@ -0,0 +1,14 @@
+
+import java.util.List;
+
+@Service
```
remove annotation

AdanAlvarezV committed:
Change service annotation

Service annotation removed on IPersonService
Service annotation added on PersonServiceImpl
